src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -31,8 +32,19 @@ public async Task GenerateSDK()
             var sw = new Stopwatch();
             // Step 1 - Read input stream
             string inputPath = config.OpenAPIFilePath;
+
+            try {
+                // doing this verification at the begining to give immediate feedback to the user
+                Directory.CreateDirectory(config.OutputPath);
+            } catch (Exception ex) {
+                logger.LogError($"Could not open/create output directory {config.OutputPath}, reason: {ex.Message}");
+                return;
+            }
+            
             sw.Start();
             using var input = await LoadStream(inputPath);
+            if(input == null)
```

Comment:
We should log this scenario where the OpenAPI spec is null.

Reply:
the LaodStream method already logs errors. I have added this bypass to avoid throwing in the LoadStream method and avoid having the GenerateSDK method continue and throw when we couldn't load the openAPI document